kyo-core/shared/src/main/scala/kyo/Log.scala

```diff
@@ -0,0 +1,123 @@
+package kyo
+
+import kyo.internal.LogPlatformSpecific
+
+object Log extends LogPlatformSpecific:
```

Comment:
I've removed the `debug` methods here. I'm planning to introduce a new `Debug` effect based on `Safepoint.Interceptor`.